portfolio/src/main/webapp/index.jsp

```diff
@@ -128,32 +128,45 @@ <h1>Work Experience</h1>
               
               <dt>Foundant Technologies</dt>
               <dt class="workposition">Spring 2020 - Client Services Intern</dt>
-              <dd>Assist clients that use Foundant’s grant and scholarship management software or fund accounting software with any technical support questions or issues they might have.</dd>
+              <dd>Assist clients that use Foundant's grant and scholarship management software or fund accounting software with any technical support questions or issues they might have.</dd>
               
               <dt>Silver Lining Entertainment</dt>
               <dt class="workposition">October 2016 to November 2018 - Talent Manager's Assistant</dt>
               <dd>Managed desk of 20+ actors and coordinated between production, publicists, casting, and clients. Logged over 2,000 company commissions and increased efficiency by digitizing the tracking system.</dd>
               
               <dt>USC Annenberg - Media, Diversity, & Social Change Initiative</dt>
               <dt class="workposition">Summer 2015 - Student Research Assistant</dt>
-              <dd>Analyzed and quantified TV & Film content metrics for the 2014-2015 study: ‘Inclusion or Invisibility? The Comprehensive Annenberg Report on Diversity in Entertainment’ as seen <a href="https://bit.ly/2oBKDUr">here.</a></dd>
+              <dd>Analyzed and quantified TV & Film content metrics for the 2014-2015 study: 'Inclusion or Invisibility? The Comprehensive Annenberg Report on Diversity in Entertainment' as seen <a href="https://bit.ly/2oBKDUr">here.</a></dd>
       </section>
 
       <!-- Contact Me Section -->
-      <section id="contactme">
+
+      <%@ page import = "com.google.appengine.api.users.UserService, com.google.appengine.api.users.UserServiceFactory" %>
```

Comment:
Unless you saw differently in a style guide, I think these imports should each get their own directive:

```
<%@ page import = "com...UserService" %>
<%@ page import = "com...UserServiceFactory" %>
```

Reply:
I think I saw an example on stackoverflow, but I can't speak for the legitimacy of it. I'll change this to two lines.